src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -332,7 +332,7 @@ bool LibraryCallKit::inline_vector_nary_operation(int n) {
 //                  int start, int step, int wrap, ShuffleIotaOperation<Sh, E> defaultImpl)
 bool LibraryCallKit::inline_vector_shuffle_iota() {
   // TODO shuffle is not supported on SVE
-  if (UseSVE > 0) {
+  if (Matcher::supports_scalable_vector()) {
     if (C->print_intrinsics()) {
       tty->print_cr("  ** shuffle iota not supported on SVE");
     }
```

Comment:
I guess these parts will be removed in future if shuffle is supported with SVE. Could you please add some comments for it?

Reply:
OK. Thank you for your review.